ccmbioinfo/lfs
# CARDS-1596: Text/Print exports expose data that shouldn't be accessible

To test:
- log in as admin
- create a new Patient Information with login credentials
- create a new Visit Information with a visit date in the future
- create new data forms
- generate a token for the visit
- in a private browsing window, log in with the token
- fill in the forms, submit
- back as admin, make sure the score and interpretations are correct
- copy the URL for each form, remove the `content.html`, append `.deep.json` or `.txt` or `.md` and open them in the patient's private browsing session
- make sure the scores and interpretations are not present for the patient
